src/utils/map.go

```diff
@@ -65,3 +67,52 @@ func convertValue(value interface{}) interface{} {
 		return value
 	}
 }
+
+// updates *orderedMap nested value stored on path and returns the updated map
+func UpdateIn(content *orderedmap.OrderedMap, keyPath KeyPath, value interface{}) *orderedmap.OrderedMap {
+	return updateInElement(content, keyPath, value).(*orderedmap.OrderedMap)
+}
```

Comment:
nakoniec som to nazval UpdateIn pretoze to vlastne vrati tu sami orderedMap, resp jej pointer.